src/averageAges.js

```diff
@@ -55,7 +70,26 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const mothers = people.filter(person => onlyWithSon
+    ? person.sex === 'f' && people.some(children =>
```

Comment:
If it's mother, it's woman too

```suggestion
    ? people.some(children =>
```